includes/class-newspack-newsletters.php

```diff
@@ -962,7 +962,8 @@ public static function branding_scripts() {
 		$screen = get_current_screen();
 		if (
 			self::NEWSPACK_NEWSLETTERS_CPT !== $screen->post_type &&
-			Newspack_Newsletters_Ads::NEWSPACK_NEWSLETTERS_ADS_CPT !== $screen->post_type
+			Newspack_Newsletters_Ads::NEWSPACK_NEWSLETTERS_ADS_CPT !== $screen->post_type &&
```

Comment:
Will the namespace change be made in another PR?

Reply:
what namespace change? In Ads?

Reply:
This is an ad feature inside Newsletters (`includes/class-newspack-newsletters-ads.php`). I assume the new namespace will be applied to all of the plugin classes eventually, correct?

Reply:
I'd love if we started using more namespaces across the codebase... and also that we relied on the autoloader so we don't have to keep `require_once` everything. But I figured this is out of the scope of this PR and this project... let's do baby steps.

BUT, what I considered doing there was to not have all these `if`s there. Instead, each of the 3 classes would enqueue the same script, each one in its own page... so we don't have this cross reference. There's a better hook to use. I could do that